tools/cldr/datefmts.js

```diff
@@ -2327,8 +2327,7 @@ module.exports = {
     writeFormats: function(outputDir, outfile, group, localeComponents) {
         var dir = path.join.apply(undefined, [outputDir].concat(localeComponents));
         var filename = path.join(dir, outfile);
-        var contents = stringify(group.data, {space: 4});
-
+        var contents = JSON.stringify(group.data, undefined, 4); // to filter out `{\n}` case
```

Comment:
The real sorting happen in line 2336. and with the `stringify(group.data, {space: 4});` I realized there's case to return  `{\n}` case, and tried to filter out the case, but did't work so I thought it's okay to leave JSON.strigify here. 
 it shouldn't be commited empty file like `en/CA/list.json` file. it created because of this line.
I think I'll rollback the list.json changes and related toolcode and tc changes. betteer to handle different PR.